### PR TITLE
:recycle: adapt to new react native API on NativeEventEmitter

### PR DESCRIPTION
**Pull request by mbouxin**
### Contributing checklist
- [x] Code must follow existing styling conventions
- [x] Added a descriptive commit message

### Solves issue(s)
#477 

Tested on RN64 but not on RN < 64

### CODE REVIEW

File: src/OT.js [hunk 1]
```diff
@@ -31,9 +31,18 @@ const checkAndroidPermissions = () => new Promise((resolve, reject) => {
 const setNativeEvents = (events) => {
   const eventNames = Object.keys(events);
   OT.setNativeEvents(eventNames);
-  each(events, (eventHandler, eventType) => {
+
+  // change in react native API : listeners does not exist any more with RN 0.64 but is replaced by listenerCount
+  let hasRegisteredEvents;
+  if (nativeEvents.listeners) {
     const allEvents = nativeEvents.listeners();
```
Comment (abdulajet) [inline src/OT.js:38]: `nativeEvents.listeners()` is the function that has been deprecated no?

Comment (mbouxin) [reply src/OT.js:38]: I added this for backward compatibility

Comment (abdulajet) [reply src/OT.js:38]: ok I think I get it, will test it this afternoon

**Comment by mbouxin** [comment]
Dev here have been put in #493

I close this PR

**Pull request closed without merging**